topic-operator/src/test/java/io/strimzi/operator/topic/BatchingTopicControllerTest.java

```diff
@@ -75,122 +70,102 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyList;
 import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verifyNoInteractions;
 
 /**
- * This test is not intended to provide lots of coverage of the {@link BatchingTopicController}
- * (see {@link TopicControllerIT} for that), rather it aims to cover some parts that a difficult
- * to test via {@link TopicControllerIT}
+ * This test is not intended to provide lots of coverage of the {@link BatchingTopicController}, 
+ * rather it aims to cover some parts that a difficult to test via {@link TopicControllerIT}.
```

Comment:
I gess we should try to define better what those parts _are_, so as to answer the question "Where should I add my test?"

Reply:
Added some use cases.